### PR TITLE
COMMERCE-1067 - headless-delivery - TEMP commit, for batch engine testing BlogPostingResourceImpl and BlogPosting are updated, how can I do the same in tests?

### PR DESCRIPTION
**Pull request by ivicac**
Best Regards

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

File: modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineTaskExecutorImpl.java [hunk 1]
```diff
@@ -121,10 +121,9 @@ private void _execute(BatchEngineTask batchEngineTask) throws Throwable {
 				if (items.size() < batchEngineTask.getBatchSize()) {
 					items.add(item);
 				}
-				else {
-					_commitItems(
-						batchEngineTaskItemWriter, items,
-						batchEngineTaskOperation);
+
```
Comment (shuyangzhou) [inline modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineTaskExecutorImpl.java]: Not sure what do you mean here, isn't that "if (!items.isEmpty())" right after this while loop is catching up the tail?

Comment (ivicac) [reply modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineTaskExecutorImpl.java]: @shuyangzhou this problem is, that last statement only saves the last batch of unsaved items, but for all batches you will loose one record per batch group

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/78512

File: modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineTaskExecutorImpl.java (hunk 1 above)
Comment (ivicac) [reply modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineTaskExecutorImpl.java]: @shuyangzhou , sorry, I see that resource instance is still created in addService method. I was misguided with the return type of addingService method. Everything is ok, and it was my mistake to not to release it in my original PR.

**Pull request closed without merging**